components/Player/Pages/playerMatchesColumns.jsx

```diff
@@ -26,43 +24,16 @@ export default [{
   sortFn: true,
   displayFn: transformations.skill,
 }, {
-  displayName: 'Mode',
-  field: 'game_mode',
-  width: 2.5,
-  sortFn: true,
-  displayFn: transformations.game_mode,
-}, {
-  displayName: 'Ended',
-  field: 'start_time',
-  width: 2,
-  displayFn: transformations.start_time,
-  sortFn: true,
-}, {
   displayName: 'Len',
   field: 'duration',
-  width: 2,
+  width: 1.5,
   sortFn: true,
   displayFn: transformations.duration,
 }, {
-  displayName: 'K',
+  displayName: 'KDA',
   field: 'kills',
-  width: 1,
-  sortFn: true,
-}, {
-  displayName: 'D',
-  field: 'deaths',
-  width: 1,
-  sortFn: true,
-}, {
-  displayName: 'A',
-  field: 'assists',
-  width: 1,
-  sortFn: true,
-}, {
-  displayName: 'P',
-  field: 'version',
-  width: 1,
-  sortFn: true,
-  displayFn: (row, col, field) => (field ? <ActionDone /> : <div />),
+  sortFn: (row) => ((row.kills + row.deaths + row.assists) / row.deaths),
+  width: 2,
```

Comment:
That seems wasteful, but I don't think it's really an issue. I would guess people care most about seeing where they have a positive impact vs. where they have a negative impact. The only case where that might be different is if they wanted to sort on all their matches to see their highest kills or deaths matches, but that seems like an edge case we don't need to support.